sdk/tables/azure-data-tables/inc/azure/data/tables/enum_operators.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <type_traits>
+
+namespace Azure { namespace Data { namespace Tables {
+  /**
+   * @brief Bitwise OR operator for enum class.
+   */
+  template <class E, class = std::enable_if_t<std::is_enum<E>{}>> E operator|(E lhs, E rhs)
+  {
+    using type = std::underlying_type_t<E>;
+    return static_cast<E>(static_cast<type>(lhs) | static_cast<type>(rhs));
+  }
+

```

Comment:
please add the `operator~`.
Unlike the `operator!`, the `~` is a bitwise one, and it is useful in expressions like clearing the flag, i.e. `flags &= ~MASK;`